src/components/FavoriteArticle.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  readTime,
+  title,
+  image,
+  name
+}) => (
+  <div className="row article-col">
+    <div className="col-6">
+      <Link to="/article/sut5-sg565">
+        <img className="article-img" src={image} alt="" />
```

Comment:
I thought we agreed on using a `background image on a container (div or span)` instead of `<img> tags`